napari/_qt/qt_viewer.py

```diff
@@ -303,8 +271,36 @@ def __init__(
 
         for layer in self.viewer.layers:
             self._add_layer(layer)
-        for overlay in self.viewer._overlays.values():
-            self._add_overlay(overlay)
+
+    @property
+    def view(self):
+        """
+        Rectangular  vispy viewbox widget in which a subscene is rendered. Access directly within the QtViewer will
+        become deprecated.
+        """
+        warnings.warn(
+            trans._(
+                "Access to napari.qt.qt_viewer.view will become deprecated in the future. Change to napari.qt.qt_viewer.canvas.view instead."
+            ),
+            FutureWarning,
+            stacklevel=2,
+        )
+        return self.canvas.view
+
+    @property
+    def camera(self):
+        """
+        The Vispy camera class which contains both the 2d and 3d camera used to describe the perspective by which a
+        scene is viewed and interacted with. Access directly within the QtViewer will become deprecated.
+        """
+        warnings.warn(
+            trans._(
+                "Access to napari.qt.qt_viewer.camera will become deprecated in the future. Change to napari.qt.qt_viewer.canvas.camera instead."
```

Comment:
```suggestion
                "Access to QtViewer.camera will become deprecated in the future. Change to QtViewer.canvas.camera instead."
```

We should use class name, not full path.